packages/client/src/components/Logo/index.css

```
@@ -0,0 +1,22 @@
+.logo {
+  display: flex;
+  flex-direction: column;
+}
+.logo-text {
+  font-family: NinjaNaruto;
```

Для шрифта есть css-переменная `--font-naruto` с дополнительными безопасными шрифтами
```suggestion
  font-family: var(--font-naruto);
```